README.md

```diff
@@ -126,6 +127,54 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`, perhaps in an initialization file.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.semian_configuration`
+that takes two parameters, `host` and `port like `127.0.0.1` and `80` or `github_com` and `80`,
+and returns two values, a `semian_identifier` and a `Hash` with the keys as follows.
+
+```ruby
+SEMIAN_OPTIONS = { tickets: 1,
+                   success_threshold: 1,
+                   error_threshold: 3,
+                   error_timeout: 10 }
+Semian::NetHTTP.semian_configuration = proc do |host, port|
+  semian_identifier = "nethttp_#{host}_#{port}"
```

Comment:
^ I changed to `nethttp` since if doesn't make more sense to have `http_` if `https_` is not there also, and that introduces two possible `semian_identifier` prefixes.

If we provide the ability to customize the `semian_identifier`, then it's easier to do work with one `proc` that outputs both the identifier and the configuration options than to have two `proc`'s that probably repeat the same `switch`-`case` logic. I guess returning a `{identifier: semian_identifier, options: SEMIAN_OPTIONS}` would be better since you *can* make the user not specify the identifier. If no one objects I'll change it to that